subsystems/DiscordRPC/src/main/java/org/terasology/engine/subsystem/discordrpc/DiscordRPCSubSystem.java

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.terasology.engine.subsystem.rpc;
+package org.terasology.engine.subsystem.discordrpc;
```

Comment:
Should this package name still have `engine` in it?
```suggestion
package org.terasology.subsystem.discordrpc;
```

Reply:
it is still part of engine yet.

Reply:
as in: still in the _engine repo_? or still technically in the engine? I don't mind too much, was just wondering...